src/Parallel/GlobalCache.hpp

```diff
@@ -14,6 +14,7 @@
 #include <vector>
 
 #include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataBox/TagTraits.hpp"
```

Comment:
In the commit message of " Add static assert that CacheTag of FromGlobalCache is a simple tag " you have `This is makes` ->`This makes`